install.sh

```diff
@@ -173,6 +173,15 @@ if [ ! -f $BUILD/progress/system-pkgs ]; then
   touch $BUILD/progress/system-pkgs
 fi
 
+# Locally mirror various third-party JavaScript, CSS, etc, so that we don't
```

Comment:
If you're going to do it this way, can we do it at build time?  This is okay if it takes a long time, but rerunning install.sh isn't very common, and if it's quick., I'd rather just do it at build.